tutorial/practicals/LFRic/building_code/1_simple_kernels/README.md

```diff
@@ -0,0 +1,162 @@
+# Tutorial 1: Create and run simple kernels
+
+In this tutorial we will learn how to
+
+* Create simple functional kernels to update LFRic fields;
+* Call the created kernels from an algorithm.
+
+Each kernel in this tutorial performs one simple mathematical operation
+in order to learn how to work with the basic building blocks of an
+[LFRic kernel](LFRic_kernel_structure.md):
+
+* [Kernel metadata](LFRic_kernel_structure.md#kernel-metadata) and
+* [Kernel subroutine](LFRic_kernel_structure.md#kernel-subroutine) including
+  - [Argument list and declarations](
+    LFRic_kernel_structure.md#argument-list-and-declarations) and
+  - [Loops](LFRic_kernel_structure.md#loops) to update LFRic field objects.
+
+The tutorial is further subdivided into two parts in order to learn how to
```

Comment:
I'd make it clearer that these *are* the two parts of the tutorial and that a user should do each in turn. I keep reading this file and wondering what I'm supposed to do.

Reply:
Thanks, I rephrased this and listed the supporting source accordingly.